modules/ocf_syslog/files/ocf.conf

```diff
@@ -15,41 +25,46 @@ template(name="ocf-remote" type="list") {
     constant(value="\n")
 }
 
-# template for errored remote log lines
-template(name="ocf-errors" type="list") {
-    constant(value="ERRORED_TAG[")
-    property(name="syslogtag")
-    constant(value="]  MESSAGE[")
-    property(name="timegenerated" dateFormat="rfc3339")
-    constant(value=" ")
-    property(name="hostname")
-    constant(value=":  ")
-    property(name="msg")
-    constant(value="]\n")
+# template for remote docker log filenames
+template(name="ocf-docker-file" type="list") {
+    constant(value="/var/log/remote/services/")
+    # in jessie, syslogtag has a weird ":" appended, so we have to regex that out;
+    # the regex bits can be removed when we're totally on stretch.
+    property(name="syslogtag" regex.Expression="^[^:]+" regex.Type="ERE" regex.Match="0" securepath="replace")
+    constant(value="/")
+    property(name="syslogtag" regex.Expression="^[^:]+" regex.Type="ERE" regex.Match="0" securepath="replace")
```

Comment:
I'm not sure how to remove this duplication, but maybe it's fine for now? I think we're actually on stretch everywhere now, so we can probably remove the more complicated regex stuff here at least, but the duplication of the property will still remain.

Reply:
Yes, we're definitely completely on stretch now for the dockers. Duplication of the property? Whatever...